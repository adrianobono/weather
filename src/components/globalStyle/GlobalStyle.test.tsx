import { GlobalStyle } from "./";

const TestRenderer = require("react-test-renderer");

test("check some styles from element", () => {
  const css = TestRenderer.create(GlobalStyle.defaultProps).toJSON();
  expect(css).toMatchSnapshot();
});
