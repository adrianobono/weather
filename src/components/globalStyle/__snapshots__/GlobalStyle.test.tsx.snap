// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check some styles from element 1`] = `null`;
